### PR TITLE
Merge 4.10.0 into 4.10.1

### PR DESCRIPTION
**Pull request by Desvelao**
Merge 4.10.0 into 4.10.1
> [!CAUTION]
> Merge PR strategy: Create a merge commit

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,8 +7,13 @@
     "templateVersion": "2.16.0"
   },
   "wazuh": {
+<<<<<<< HEAD
     "version": "4.10.1",
     "revision": "00"
+=======
+    "version": "4.10.0",
+    "revision": "06"
+>>>>>>> a05a8dedd2ca5e085813ed094703a3d0db21191f
```
Comment (guidomodarelli) [inline package.json:16]:
```suggestion
    "version": "4.10.0",
    "revision": "06"
```

File: .github/workflows/dev-environment.yml [hunk 2]
```diff
@@ -75,22 +75,24 @@ jobs:
               yarn config set registry https://registry.yarnpkg.com;
               cd /home/node/kbn/plugins/wazuh-security-plugin && yarn && ${{ inputs.command }};
             '
-      - name: Get the plugin version
+      - name: Get the plugin version and and format reference name
```
Comment (guidomodarelli) [inline .github/workflows/dev-environment.yml:78]:
```suggestion
      - name: Get the plugin version and format reference name
```

**Review by guidomodarelli — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Desvelao) [reply package.json]: Fixed

File: .github/workflows/dev-environment.yml (hunk 2 above)
Comment (Desvelao) [reply .github/workflows/dev-environment.yml]: Fixed

**Review by guidomodarelli — APPROVED** [review]
Legend:
⚫ 🠆 The test hasn't started yet.
🟢 🠆 All checks passed.
🟡 🠆 There is at least one expected fail or skipped test and no failures.
🔴 🠆 There is at least one failed check.
⚪ 🠆 Doesn't apply.
🔧 🠆 Request changes. 

CR 🟢

**Pull request merged**